.github/workflows/inspecjs-tests.yml

```diff
@@ -12,6 +12,9 @@ jobs:
 
     steps:
      - uses: actions/checkout@v2
+     - uses: actions/setup-ruby@v1
+       with:
+         ruby-version: '2.7.4'
```

Comment:
`ruby/setup-ruby` seems to be the officially supported action, this one is deprecated.

Reply:
Also, I think ruby 2.7 without specifying the 4 is sufficient, that way it will always run against the latest security patch of ruby 2.7.